yt_dlp/extractor/instagram.py

```diff
@@ -403,17 +420,6 @@ def _real_extract(self, url):
             self.report_warning('General metadata extraction failed (some metadata might be missing).', video_id)
         media.update(traverse_obj(general_info, ('data', 'shortcode_media')) or {})
 
-        if self._get_cookies(url).get('sessionid'):
-            info = traverse_obj(self._download_json(
-                f'{self._API_BASE_URL}/media/{_id_to_pk(video_id)}/info/', video_id,
-                fatal=False, note='Downloading video info', headers={
-                    **self._API_HEADERS,
-                    'X-CSRFToken': csrf_token_value,
-                }), ('items', 0))
-            if info:
-                media.update(info)
-                return self._extract_product(media)
-
         webpage, urlh = self._download_webpage_handle(url, video_id)
```

Comment:
requesting the regular webpage has caused a regression, see: https://github.com/yt-dlp/yt-dlp/issues/4750#issuecomment-1226144212